snmp/datadog_checks/snmp/config.py

```diff
@@ -37,7 +37,7 @@ def __init__(self, instance, warning, log, global_metrics, mibs_path, profiles,
         if profile:
```

Comment:
Could you add to desc what motivates this change?

Reply:
Updated the PR desc.

Reply:
thx